CHANGES.rst

```diff
@@ -1,10 +1,12 @@
 Changelog
 =========
 
-4.2.3 (unreleased)
+4.3.0 (unreleased)
 ------------------
 
-- TBD
+- Drop support for Python 2.6 and 3.2.
+
+- Claim support for Python 3.5.
```

Comment:
Prefer "Add support" to "Claim support":  there *isn't* support until we test for it.